### DISCUSSION TITLE
Cache regenerating

### DISCUSSION THREAD

**Discussion by c2ofh** [fnando/i18n-js#652] [Q&A] [answered]
Hi,

we have the problem that the cache file (tmp/cache/i18n-js.yml ) is regenerated very often. What is the reason here?
Due to the regenerate we often have the error that the file is briefly not available or the built was not finished and then requests occuring errors, because the YML can not be parsed.

Is there any way that the cache stays persistent, because only on a deploy changes can be made to the files needed to generate the translations.js.

**Comment by PikachuEXE** [comment, marked as answer]
I assume you are using the middleware
You can see the implementation here
https://github.com/fnando/i18n-js/blob/v3/lib/i18n/js/middleware.rb

Since I don't implement nor use that feature I am unable to explain the reason(s)

**Reply by c2ofh** [reply]
Thanks this does the trick :)
